pkgs/development/compilers/temurin-bin/jdk-darwin-base.nix

```diff
@@ -19,7 +19,7 @@ let
   providedCpuTypes = builtins.filter (arch: builtins.elem arch validCpuTypes) (
     builtins.attrNames sourcePerArch
   );
-  result = stdenv.mkDerivation {
+  result = stdenv.mkDerivation rec {
```

Comment:
Be careful with `rec` here. If a user overrides a version on this derivation it won't properly update throughout the derivation, as far as I am aware.

Reply:
I always forget that I'm supposed to use `finalAttrs` instead. Updated